components/authz-service/storage/v1/storage.go

```diff
@@ -300,6 +300,14 @@ func DefaultPolicies() (map[string]*Policy, error) {
 			Effect:   "allow",
 			Version:  1,
 		},
+		constants.ApplicationsServiceGroupsPolicyID: {
```

Comment:
You've added this policy to our system policies, which are hidden policies that govern permissions that Automate needs to work correctly (i.e. every user always needs to be able to get the license status so they won't be always blocked by the license modal). These policies never change.

My guess is that we should be able to change permissions on this API. In that case, instead of adding this policy, we'll want to [add the API's v2 action to the role(s)](https://github.com/chef/automate/blob/master/components/authz-service/storage/v2/storage.go#L218) that should have access. Maybe Viewer and Editor?

Reply:
actually just kidding about that system policy comment! i misread the code. this is correct for adding a v1 default policy 👍 

for v2 permissions:
1. add `applications:*` to the [Editor role](https://github.com/chef/automate/blob/master/components/authz-service/storage/v2/storage.go#L229)
2. add `applications:serviceGroups:list` to the [Viewer role](https://github.com/chef/automate/blob/master/components/authz-service/storage/v2/storage.go#L244)
3. add a datamigration updating those two roles with their new actions (similar to [this migration](https://github.com/chef/automate/blob/master/components/authz-service/storage/postgres/datamigration/sql/06_v2_roles.up.sql))

Reply:
If we make changes to these roles please also update the iam v2 docs, thx!